core/common/src/main/java/alluxio/util/CommonUtils.java

```
@@ -383,34 +381,8 @@ public static Throwable getRootCause(Throwable e) {
   public static void closeQuietly(Closer closer) {
     try {
       closer.close();
-    } catch (Exception e) {
-      // Ignore.
-    }
-  }
-
-  /**
-   * Closes a closer, converting any IOException to an {@link AlluxioStatusException}.
-   *
-   * @param closer the closer
-   */
-  public static void close(Closer closer) {
-    try {
-      closer.close();
-    } catch (IOException e) {
-      throw AlluxioStatusException.fromIOException(e);
-    }
-  }
-
-  /**
-   * Closes a Closeable, converting any IOException to an {@link AlluxioStatusException}.
-   *
-   * @param closeable the Closeable
-   */
-  public static void close(Closeable closeable) {
-    try {
-      closeable.close();
     } catch (IOException e) {
-      throw AlluxioStatusException.fromIOException(e);
+      // Ignore.
```

I've removed the method so that we don't completely swallow exceptions.